app/assets/javascripts/mobile.js

```diff
@@ -30,10 +31,18 @@ $(document).ready(function(){
 
   /* Drawer menu */
   $('#menu_badge').bind("tap click", function(evt){
-    evt.preventDefault();
     $("#app").toggleClass('draw');
   });
 
+  var drawerSlide = new Hammer(document.getElementById('app'));
```

Comment:
You are right, maybe the whole library isn't needed: but it's not that big of a library, (17k uncompressed, 4k compressed), and the code has to be written: it's better that somebody else deals with all sort of bugs and quirks.

Also, maybe we can add other gestures later, so it may be useful.

I did test with Firefox for Android. Will test in the simulator soon.  In Firefox with the Responsive Design Mode activated it works fine.